app/Models/Equipment/BreakageImage.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Models\Equipment;
+
+use Illuminate\Database\Eloquent\Model;
+
+class BreakageImage extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     */
+    protected $fillable = [
+        'filename',
+        'mime',
+        'report_id',
```

Comment:
nit: it's generally recommended that FK names use the table name, eg `equipment_breakage_id`